src/Pages/Patients/AllPatients/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import { useI18n } from "providers/I18n"
+
+AllPatients.propTypes = {}
+
+function AllPatients() {
+  const { t } = useI18n()
+  return (
+    <div>
+      <h1>{t("int.all-patients-header", "patients")}</h1>
```

Comment:
This should produce a warning in the console. Open your DevTools and check it. 
Remove this warning by adding the `int.all-patients-header` key to the src/providers/I18n/translations.json for both en and el